### PR TITLE
add option to set minKubeVersion

### PR DESCRIPTION
**Pull request by fromanirh**
add the option to set minKubeVersion, and do some cleanup along the way.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
/retest
seems transient error

**Comment by fromanirh** [comment]
/hold cancel
previous concerns about changing the pullspec URLs where resolved (mostly it was me *not* having called `make generate` properly)

File: Makefile [hunk 1]
```diff
@@ -188,7 +188,7 @@ deploy: cluster-deploy
 	# TODO - deprecated, will be removed soon in favor of cluster-deploy
 
 .PHONY: cluster-deploy
-cluster-deploy:
+/luster-deploy:
```
Comment (cynepco3hahue) [inline Makefile:191]: typo

Comment (fromanirh) [reply Makefile:191]: great cach, thanks!

**Comment by fromanirh** [comment]
/retest
it seems CI healed (this time for real?)

**Pull request merged**